/**:
  ros__parameters:
  # Kvaser node parameters
    hardware_id: 42774
    circuit_id: 0
    bit_rate: 500000
    enable_echo: false
  # DBW CAN node
    max_steer_angle: 470.0
